src/vega.schema.ts

```diff
@@ -180,3 +180,154 @@ export interface VgImputeTransform {
   method?: 'value' | 'median' | 'max' | 'min' | 'mean';
   value?: any;
 }
+
+export interface VgAxisBase {
```

Comment:
Add description -- something like:

/**
 * Base object for Vega's Axis and Axis Config.  
 * All of these properties are both properties of Vega's Axis and Axis Config.  
 */